gc.rb

```diff
@@ -196,6 +196,24 @@ def self.stat hash_or_key = nil
     Primitive.gc_stat hash_or_key
   end
 
+  # :nodoc:
+  # call-seq:
+  #    GC.stat_size_pool(size_pool_idx) -> Hash
+  #    GC.stat_size_pool(size_pool_idx, hash) -> Hash
+  #    GC.stat_size_pool(size_pool_idx, :key) -> Numeric
+  #
+  # Returns a Hash containing information about a size pool in the GC.
+  #
+  # The contents of the hash are implementation specific and may be changed in
+  # the future.
+  #
+  # If the optional argument, hash, is given, it is overwritten and returned.
+  #
+  # This method is only expected to work on C Ruby.
```

Comment:
```suggestion
  # This method is only expected to work on CRuby.
```
Conform to usual branding.